### PR TITLE
[SLATE] Bump version to 2023.11.05

### PR DESCRIPTION
**Pull request by jeremiahpslewis**

### CODE REVIEW

File: S/SLATE/build_tarballs.jl [hunk 1]
```diff
@@ -20,18 +19,27 @@ script = raw"""
...
-  -DMPI_RUN_RESULT_CXX_libver_mpi_normal="0" \
-  -DMPI_RUN_RESULT_CXX_libver_mpi_normal__TRYRUN_OUTPUT="" \
-  -Drun_result="0" \
-  -Drun_result__TRYRUN_OUTPUT="ok" \
-  -Dc_api=yes \
-  ..
+
+CMAKE_FLAGS=()
+CMAKE_FLAGS=()
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix})
```
Comment (giordano) [inline S/SLATE/build_tarballs.jl:25]: Why not
```suggestion
CMAKE_FLAGS=(-DCMAKE_INSTALL_PREFIX=${prefix})
```
?

Also why not a single `CMAKE_FLAGS` setting?

File: S/SLATE/build_tarballs.jl [hunk 2]
```diff
@@ -20,18 +19,27 @@ script = raw"""
...
+CMAKE_FLAGS+=(-DMPI_RUN_RESULT_CXX_libver_mpi_normal__TRYRUN_OUTPUT="")
+CMAKE_FLAGS+=(-Drun_result="0")
+CMAKE_FLAGS+=(-Drun_result__TRYRUN_OUTPUT="ok")
+CMAKE_FLAGS+=(-Dblas_complex_return=return)
+
+if [[ ${nbits} == 32 ]]; then
+    CMAKE_FLAGS+=(-Dblas_int=int32)
+else
+    CMAKE_FLAGS+=(-Dblas_int=int64)
+fi
```
Comment (giordano) [inline S/SLATE/build_tarballs.jl:40]: Why not
```suggestion
CMAKE_FLAGS+=(-Dblas_int=int${nbits})
```
?

File: S/SLATE/build_tarballs.jl [hunk 3]
```diff
@@ -20,18 +19,27 @@ script = raw"""
...
+
+cmake ${CMAKE_FLAGS[@]} ..
```
Comment (giordano) [inline S/SLATE/build_tarballs.jl:42]:
```suggestion
cmake "${CMAKE_FLAGS[@]}" ..
```

File: S/SLATE/build_tarballs.jl [hunk 4]
```diff
@@ -50,7 +58,10 @@ platforms, platform_dependencies = MPI.augment_platforms(platforms)
 
 # Avoid platforms where the MPI implementation isn't supported
 # OpenMPI
-platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p) == "glibc"), platforms)
+platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l"), platforms)
+platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv7l"), platforms)
+platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "i686"), platforms)
+
```
Comment (giordano) [inline S/SLATE/build_tarballs.jl:64]: https://github.com/JuliaPackaging/Yggdrasil/blob/4e817d19789e974c44ec1ecd233bc82d18d4ca70/H/HDF5/build_tarballs.jl#L245

File: S/SLATE/build_tarballs.jl [hunk 5]
```diff
@@ -69,4 +70,4 @@ append!(dependencies, platform_dependencies)
 
 # Build the tarballs.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               augment_platform_block, julia_compat="1.6", preferred_gcc_version=v"7")
+               augment_platform_block, julia_compat="1.6", preferred_gcc_version = v"11.1.0")
```
Comment (giordano) [inline S/SLATE/build_tarballs.jl:73]: https://github.com/JuliaPackaging/Yggdrasil/blob/cace4d2d7cf0b955d0bf54631a3b832050c8cb53/CONTRIBUTING.md#compatibility-tips

Comment (jeremiahpslewis) [inline S/SLATE/build_tarballs.jl:73]:
```suggestion
               augment_platform_block, julia_compat="1.6", preferred_gcc_version = v"7")
```

**Pull request merged**